src/sndfile.c

```diff
@@ -325,7 +325,7 @@ static char	sf_syserr [SF_SYSERR_LEN] = { 0 } ;
 
 SNDFILE*
 sf_open	(const char *path, int mode, SF_INFO *sfinfo)
-{	SF_PRIVATE 	*psf ;
+{	SF_PRIVATE	*psf ;
```

Comment:
Seems to be a whole bunch of semantically null changes in this PR. They should be removed.